util/seed/user.go

```
@@ -20,6 +20,8 @@ var osoba *model.User
 var osoba2 *model.User
 var admin *model.User
 var hak *model.User
+var mup *model.User
```

golangci-lint (1.64.8)

var `mup` is unused

(unused)